package.json

```diff
@@ -11,13 +11,25 @@
     "build": "grunt",
     "css": "grunt css",
     "watch": "grunt watch",
-    "deploy": "grunt deploy"
+    "deploy": "grunt deploy",
+    "e2e": "jest",
+    "wp-env": "wp-env"
+  },
+  "jest": {
+    "preset": "jest-puppeteer",
+    "setupFilesAfterEnv": [
+      "expect-puppeteer"
+    ],
+    "globals": {
+      "STORE_URL": "http://localhost:8801"
```

Comment:
You probably want to expose this as an env variable instead of a global, globals are bits of code that should be accessed anywhere (`wp`, `wc`, ...).

Reply:
Agreed - this is a bit temporary. 

For now I'm happy with this, as env vars are more global. But I agree we will likely iterate on this and it may well end up in an env var if needed.

Reply:
Using a `.env` file is handy.

Reply:
Agreed! Does wp-env support these too? I guess maybe it does if it's docker-compose under the hood ..

Reply:
> Does wp-env support these too?

`wp-env` does not support `.env` files. But something like [`dotenv`](https://www.npmjs.com/package/dotenv) makes it fairly trivial to support `.env` files in any other scripts we do (hence the suggestion).